stade/core/templates/wizard/create-submission.html

```diff
@@ -69,7 +69,48 @@ <h2>Upload Submission Data</h2>
 ISIC_0001102,0.54389,0.64057,0.61898,0.554,0.2462,0.0586,0.4182,0.124,0.6313</pre>
 
     <p>For more information on submitting, please see the <a target="_blank" href="https://challenge2019.isic-archive.com/submit.html">submission documentation</a>.</p>
-    {% endif %}
+        {% elif approach.task.id == 56 %}
```

Comment:
Could you leave a comment to indicate what `56` is?
```suggestion
        {# Task 56 is 2020 Live #}
        {% elif approach.task.id == 56 %}
```